### PR TITLE
Fixes issues with empty polys

### PR DESCRIPTION
**Pull request by jlaura**

### CODE REVIEW

File: autocnet/cg/cg.py [hunk 1]
```diff
@@ -381,18 +381,18 @@ def distribute_points_in_geom(geom,
     """
     coords = list(zip(*geom.envelope.exterior.xy))
     short = np.inf
-    lng = -np.inf
+    long = -np.inf
```
Comment (jlaura) [inline autocnet/cg/cg.py]: @jessemapel :heart:

File: autocnet/spatial/overlap.py [hunk 2]
```diff
@@ -70,13 +70,12 @@ def place_points_in_overlaps(nodes, size_threshold=0.0007,
...
                                               iterative_phase_kwargs=iterative_phase_kwargs,
                                               distribute_points_kwargs=distribute_points_kwargs))
-
     Points.bulkadd(points)
 
 def cluster_place_points_in_overlaps(size_threshold=0.0007,
                                      iterative_phase_kwargs={'size':71},
                                      distribute_points_kwargs={},
-                                     walltime='00:10:00', cam_type="csm"):
+                                     Gwalltime='00:10:00', cam_type="csm"):
```
Comment (Kelvinrr) [inline autocnet/spatial/overlap.py]: Gwalltime?

Comment (jlaura) [reply autocnet/spatial/overlap.py]: Fixed

File: autocnet/cg/cg.py [hunk 3]
```diff
@@ -344,8 +344,8 @@ def distribute_points(geom, nspts, ewpts):
     return valid
 
 def distribute_points_in_geom(geom,
-                              nspts_func=lambda x: int(round(x,1)*10),
-                              ewpts_func=lambda x: int(round(x,1)*5)):
+                              nspts_func=lambda x: ceil(round(x,1)*10),
+                              ewpts_func=lambda x: ceil(round(x,1)*5)):
```
Comment (jessemapel) [inline autocnet/cg/cg.py]: do we need round if we're doing ceil?

File: autocnet/cg/cg.py [hunk 4]
```diff
@@ -421,10 +422,11 @@ def distribute_points_in_geom(geom,
...
         nspts = ewpts_func(short)
-        ewpts = nspts_func(lng)
+        ewpts = nspts_func(long)
         if nspts == 1 and ewpts == 1:
             valid = single_centroid(geom)
         else:
             valid = distribute_points(geom, nspts, ewpts)
-
+    else:
+        print('WTF Willy')
```
Comment (jessemapel) [inline autocnet/cg/cg.py]: I like this new way to type `raise Exception`

**Pull request merged**